js/ui/status/automaticUpdates.js

```diff
@@ -36,9 +36,9 @@ const MessageTray = imports.ui.messageTray;
 const PanelMenu = imports.ui.panelMenu;
 const PopupMenu = imports.ui.popupMenu;
 
+const EOS_SETTING_AUTOMATIC_UPDATES_NOTIFICATION_TIME = "eos.automatic-updates-notification-time";
```

Comment:
Why the sudden change to `eos` here? Does that mean that `connection.is-new-connection` was supported by NM upstream? (If that's the case that would be preferred, of course)